src/avatar/Avatar.js

```diff
@@ -105,10 +102,7 @@ const AvatarComponent = ({
       {...attributes}
     >
       <Image
-        placeholderStyle={StyleSheet.flatten([
-          placeholderStyle,
-          hidePlaceholder && { backgroundColor: 'transparent' },
-        ])}
+        placeholderStyle={StyleSheet.flatten([placeholderStyle])}
```

Comment:
Did you run the changes locally?

Reply:
Pass all the tests locally.

Reply:
Tests were not covering that issue, that's why it never surfaced.

Reply:
How can I run changes locally? 
Only tests are given in the contribution guide.

Reply:
You can use this demo app -> https://github.com/react-native-elements/react-native-elements-app
After that link, the package locally using `npm link`.
Then make the required changes, it will be reflected in the app.

Reply:
Okay